### ISSUE TITLE
Responsive descriptions ready for implementation 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#35]
Responsive descriptions are also ready implementation, they are near the bottom of the [design doc](https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.m96bq8cv9p9v) everything in red is ready to go and I tried to make the examples for both qualitative and quantitative as easy as possible. Feel free to slack me any questions or confusing things.

**Ready for implementation** 
- [x] Qualitative responses
- [x] Quantitative responses

**Comment by jessegreenberg** [comment]
Alright, these have been added @Matthew-Moore240 and are ready for review. Reassigning to you.

**Comment by Matthew-Moore240** [comment]
Reviewed both, the quantitative responses sound good.
The qualitative responses sound good but I think there are two responses that could be a bit more spread out over the region.

This response I think fires to close to the middle. 
**{{right/left}} {{object}} {{somewhat close to the top}}**

The same for this response it fires a a bit to close to the middle.
**{{right/left}} {{object}} {{somewhat close to the bottom}}**

Thoughts on moving these a bit further from the middle @jessegreenberg  and @zepumph

**Comment by emily-phet** [comment]
@Matthew-Moore240 Just a heads up, the alternative input experience is going to change as we refine - see #44 and #47. Please consider if changes to how the markers move with keyboard input impacts the responsive descriptions. Perhaps your comment above is about region boundaries, which might not be impacted.

**Comment by jessegreenberg** [comment]
In a design meeting today 5/13/20 we discussed that we wanted to change the hands to sliders in the PDOM. This would involve using aria-valuetext instead of aria-live to provide updates about hand positions, and this content is read a bit differently with a screen reader.

So the current alerts will need to be re-worked a bit to be used in this context.

**Comment by terracoda** [comment]
@emily-phet and @Matthew-Moore240, it's beneficial to keep the main checklist up to date as the design progresses, see issue #11. It really helps at the end. I'll add the link to the design doc's Responsive Description in issue #11.

**Comment by zepumph** [comment]
>In a design meeting today 5/13/20 we discussed that we wanted to change the hands to sliders in the PDOM. This would involve using aria-valuetext instead of aria-live to provide updates about hand positions, and this content is read a bit differently with a screen reader.

The conversion has been done, see #47. I think we should continue that description piece over in #49, adding aria-valuetext to sliders. 

>@emily-phet and @Matthew-Moore240, it's beneficial to keep the main checklist up to date as the design progresses, see issue #11. It really helps at the end. I'll add the link to the design doc's Responsive Description in issue #11.

I agree! I was helpful for me to see your progress through it too. 

I'll keep assigned @Matthew-Moore240 so he can organize the description design issues as desired.

**Comment by jessegreenberg** [comment]
@Matthew-Moore240 and I met to discuss this today and came up with the following proposal to support aria-valuetext:
- The "On region change" portion of https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.m96bq8cv9p9v can become the aria-valuetext for the slider.
- We can remove direction and movement alerts like "Moving {{object}} up." and "Up." because this is not necessary to describe for a vertical slider.
- The "Close or around ratio" portion becomes the alert. And instead of just alerting this when we get "close or around proportion", we will alert the ratio fitness every movement.

We are going to wait until after todays design meeting to proceed. We also didn't discuss how this strategy might work with simultaneous hand movement, so that is another thing to consider.

**Comment by jessegreenberg** [comment]
During alternative input meeting today, it seemed like the relative distance was an important thing to include in the alerts.

It seems like the various input strategies will have a large impact on how alerts are designed. If we add more controls for relative distance and hand positioning that may or may not change the strategy for alerts.  So I would recommend we put this on hold until #44 is more finalized @Matthew-Moore240 and @emily-phet, but let me know if you disagree.

**Comment by emily-phet** [comment]
@Matthew-Moore240 - Can you take a look at @jessegreenberg question, and see if this still seems relevant now, or in the future?

**Comment by Matthew-Moore240** [comment]
@jessegreenberg Are we talking about whether or not we should have responses that revolve around how close or far you are to individual sliders IE: Right hand is slight above left hand

**Comment by jessegreenberg** [comment]
That is correct @Matthew-Moore240, in the design meeting on https://github.com/phetsims/ratio-and-proportion/issues/35#issuecomment-631007245, it was discussed that content like "Right hand is slight above left hand" would be important to include.

**Comment by emily-phet** [comment]
Overhauling responses, to be more like GFL:B approach. Closing this and will open new issue with specifics of that approach.
